tests/integration/github/test_pull_requests.py

```diff
@@ -87,6 +87,7 @@ def test_pr_create_upstream_upstream(self):
 
         assert pr_upstream_upstream.title == "test: upstream <- upstream"
         assert pr_upstream_upstream.status == PRStatus.open
+        assert pr_upstream_upstream.target_project.is_fork == False
```

Comment:
```suggestion
        assert not pr_upstream_upstream.target_project.is_fork
```

also flake from pre-commit doesn't like it the way it is now ;)

Reply:
😅